File: TP2 java/java/Produk.java
public class Produk {
    // atribut produk
    protected String id;
    protected String nama;
    protected int harga;

    //constructor nilai awal
    public Produk() {
        this.id = "";
        this.nama = "";
        this.harga = 0;
    }
    
    //constructor pengisian data
    public Produk(String id, String nama, int harga) {
        this.id = id;
        this.nama = nama;
        this.harga = harga;
    }

    // getter untuk ambil nilai
    public String getId() { return id; }
    public String getNama() { return nama; }
    public int getHarga() { return harga; }
}

File: TP2 php/PHP/Produk.php
<?php
class Produk {
    protected $id;
    protected $nama;
    protected $harga;

    public function __construct($id, $nama, $harga) {
        $this->id = $id;
        $this->nama = $nama;
        $this->harga = $harga;
    }

    public function getId() { return $this->id; }
    public function getNama() { return $this->nama; }
    public function getHarga() { return $this->harga; }
}
